docs/dev/error-messages.md

```diff
@@ -241,6 +241,28 @@ This isn't always restricted to the tests, either; an app under test which consu
 
 Break out your long tests into shorter tests and/or make sure that your tests are not filling up a lot of disk space on the VM.
 
+
+### Unable to Find Device Within 15 Minutes
+
+**Description**
+
+This timeout occurs most frequently when you include a specific device in your test case and that device isn’t available within 15 minutes.
+
+**Cause(s)**
+
+Our public device pool is available to all subscribed Sauce users and (as you might imagine), some devices are more popular than others. We have over 280 device configurations and thousands of devices hosted in our data center, but sometimes a test queue will build up for the most popular devices.
+
+**How to Resolve**
+
+Instead of passing a specific `deviceName`:
+
+- When you select a device from the public pool, use `deviceName` to [select the device dynamically](/mobile-apps/supported-devices/#dynamic-device-allocation). This way you can specify the type of device (make, model, OS) instead of a specific device, which increases the likelihood of finding an appropriate device that is available for your test to execute on.
+- If you are looking for a specific OS version instead of a specific make and model, you can use the [`appium:appiumplatformversion`](/dev/test-configuration-options/#appium) option to fetch a device with that OS regardless of make and model.
```

Comment:
```suggestion
- If you are looking for a specific OS version instead of a specific make and model, you can use the [`appium:platformversion`](/dev/test-configuration-options/#appium) option to fetch a device with that OS regardless of make and model.
```